kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -352,8 +352,13 @@ impl<'tcx> GotocCtx<'tcx> {
         }
 
         if let Some(stripped) = intrinsic.strip_prefix("simd_shuffle") {
-            let n: u64 = stripped.parse().unwrap();
-            return self.codegen_intrinsic_simd_shuffle(fargs, p, cbmc_ret_ty, n);
+            let _n: u64 = stripped.parse().unwrap();
+            return unstable_codegen!(self.codegen_intrinsic_simd_shuffle(
+                fargs,
+                p,
+                cbmc_ret_ty,
+                n
```

Comment:
Should this be `_n`

Reply:
I had to add `_` to all non-used variables, functions or macros because it would introduce a warning otherwise (the `unstable_codegen` macro causes them to not be used). I want to avoid "using" them because the changes would be untested. Of course, the compiler will let us know what variables have to be corrected when we restore them.